### PR TITLE
Content Updates: AO & Other Employees Pages

### PR DESCRIPTION
**Pull request by Katherine-Osos**
#  #

## 🗣 Description ##

Updated content on the Authorizing Official and Other Employees pages in the registrar.

## 💭 Motivation and context ##
Resolves https://github.com/cisagov/getgov/issues/783
Resolves https://github.com/cisagov/getgov/issues/779

### CODE REVIEW

**Review by h-m-f-t — APPROVED** [review]
v v nice

File: src/registrar/templates/application_authorizing_official.html [hunk 1]
```diff
@@ -6,13 +6,13 @@ <h2 class="margin-bottom-05">
...
 
-<p>Your authorizing official is the person within your organization who can authorize your domain request. This is generally the highest-ranking or highest-elected official in your organization.</p>
+<p>Your authorizing official is the person within your organization who can authorize your domain request. This person must be in a role of significant, executive responsibility within the organization.</p>
 
   <div class="ao_example">
     {% include "includes/ao_example.html" %}
   </div>
 
-<p>We might contact your authorizing official, or their office, to double check that they approve this request. Read more about <a href="{% public_site_url 'domains/eligibility/#you-must-have-approval-from-an-authorizing-official-within-your-organization' %}">who can serve as an authorizing official</a>.</p>
+<p>We typically don’t reach out to the authorizing official, but if contact is necessary, our practice is to coordinate with you, the requestor. Read more about <a href="{% public_site_url 'domains/eligibility/#you-must-have-approval-from-an-authorizing-official-within-your-organization' %}">who can serve as an authorizing official</a>.</p>
```
Comment (gabydisarli) [inline src/registrar/templates/application_authorizing_official.html:15]: Tiny suggestion

> We typically don’t reach out to the authorizing official, but if contact is necessary, our practice is to coordinate `first` with you, the requestor

File: src/registrar/templates/application_other_contacts.html [hunk 2]
```diff
@@ -2,7 +2,13 @@
 {% load static field_helpers %}
 
 {% block form_instructions %}
-  <p>We’d like to contact other employees in your organization about your domain request. For example, they could be involved in managing your organization or its technical infrastructure. <strong>This information will help us assess your eligibility for a .gov domain.</strong> These contacts should be in addition to you and your authorizing official. They should be employees of your organization.</p>
+  <p>To help us assess your eligibility for a .gov domain, please provide contact information for other employees from your organization.
+    <ul class="usa-list">
+      <li>They should be clearly and publicly affiliated with your organization and familiar with your domain request. </li>
+      <li>They don't need to be involved with the technical management of your domain (although they can be). </li>
+      <li>We typically don’t reach out to these employees, but if contact is necessary, our practice is to coordinate with you, the requestor. </li>
```
Comment (gabydisarli) [inline src/registrar/templates/application_other_contacts.html:9]: Same comment here if the suggestion is agreed upon

**Review by gabydisarli — APPROVED** [review]
I left a tiny suggestion, but feel free to pass on it since you are the content wizard. Looks great!

File: src/registrar/templates/application_authorizing_official.html (hunk 1 above)
Comment (Katherine-Osos) [reply src/registrar/templates/application_authorizing_official.html]: updated!

**Pull request merged**